src/services/users.service.js

```diff
@@ -0,0 +1,47 @@
+import { User } from '../models/user.model.js';
+
+async function consumeActivationToken(user) {
+  user.activationToken = null;
+  await user.save();
+}
+
+async function getAll() {
+  return User.findAll({
+    attributes: ['id', 'name', 'email', 'activationToken'],
```

Comment:
Including 'activationToken' in the attributes for 'getAll' might expose sensitive information. Consider removing it unless it's necessary for the functionality.